stdlib/2and3/logging/handlers.pyi

```diff
@@ -29,7 +29,7 @@ class WatchedFileHandler(Handler):
 
 if sys.version_info >= (3,):
     class BaseRotatingHandler(FileHandler):
-        namer = ...  # type: Optional[Callable[[str], None]]
+        namer = ...  # type: Callable[[str], str]
```

Comment:
Why did you remove the `Optional`? From https://github.com/python/cpython/blob/master/Lib/logging/handlers.py it looks like it is indeed allowed to be None.